department-of-veterans-affairs/va.gov-cms
# VACMS-7071 update section dashboard in claro

## Description

Relates to #7071. Background context: the "view" page for taxonomy terms is handled by the default theme. So some changes related to the layout are handled in the existing module, to keep things consistent (but also allow us to update the template in vagovclaro). Basically swapped the flex display logic for some grid css instead, and put all blocks into the same layout container, instead of 2 separate 3 column modules, which caused some funky layouts at certain screen widths. I exported config related to the layout builder update. Otherwise, visual changes for the blocks themselves are in the theme. Existing users shouldn't notice a difference visually.

## Testing done
Visited `/section/veterans-health-administration/vet-centers/escanaba-vet-center` as an editor (kitty.walsh) and as an admin. Looked at this page with both vagovclaro and vagovadmin as the default theme.

## Screenshots
Before (as seen on prod currrently):
<img width="836" alt="Screen Shot 2021-12-27 at 10 36 08 AM" src="https://user-images.githubusercontent.com/11279744/147502086-e1d508c9-56f1-42b0-94ef-e74f28752a9b.png">
<img width="1660" alt="Screen Shot 2021-12-27 at 10 36 18 AM" src="https://user-images.githubusercontent.com/11279744/147502089-06c2d4f7-e902-46dc-a0c6-57c2086f2aaf.png">


Updated in vagovclaro:
<img width="1915" alt="Screen Shot 2021-12-27 at 11 43 08 AM" src="https://user-images.githubusercontent.com/11279744/147502619-8a28990c-aa56-4909-81a7-3f9f3669c382.png">
<img width="936" alt="Screen Shot 2021-12-27 at 11 43 25 AM" src="https://user-images.githubusercontent.com/11279744/147502623-9794b478-a1d4-4aaa-8de1-05352cabcc58.png">
<img width="607" alt="Screen Shot 2021-12-27 at 11 43 37 AM" src="https://user-images.githubusercontent.com/11279744/147502625-d3a554f2-e81e-44ae-ad41-bceaab33bc07.png">


Updated in vagovadmin:
<img width="1913" alt="Screen Shot 2021-12-27 at 11 44 43 AM" src="https://user-images.githubusercontent.com/11279744/147502726-c2f15964-234c-4c1e-923d-868d81f169d1.png">
<img width="1056" alt="Screen Shot 2021-12-27 at 11 45 24 AM" src="https://user-images.githubusercontent.com/11279744/147502728-595cb713-332c-44c4-9616-709e643902fb.png">
<img width="299" alt="Screen Shot 2021-12-27 at 11 45 35 AM" src="https://user-images.githubusercontent.com/11279744/147502731-4c4fcddc-c8b4-4678-9411-f79464dbfac7.png">


## QA steps

1. As editor user (kitty.walsh) log in and see that vet center dashboard behaves the same as prod. Cards should be in the same order, no real visual changes.
2. log out
3. log in as Admin user. Change default theme to vagovclaro, visit /section/veterans-health-administration/vet-centers/escanaba-vet-center and notice that cards are visually updated, layout is consistent as before.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

As the visual changes are contained to the vagovclaro theme, no announcement should be necessary at this time.
